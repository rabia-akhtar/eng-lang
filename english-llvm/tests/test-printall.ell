/*
 * Test for print_all function.
 *
 * print_all should print int, double, float, boolean, and binary and single operators.
 */

int main()
{
  print_all("Hello World");
  print_all(7);
  print_all(100.98);
  print_all(true);
  print_all(-1);
  print_all(100+1);
  print_all('a');
  return 0;
}
