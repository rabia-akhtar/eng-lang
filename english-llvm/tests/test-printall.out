Hello World
7
100.980000
1
-1
101
